### DISCUSSION TITLE
index.ts (서버) 파일에서 World.multiplay폴더안에 다른 파일을 불러올수 있나요?

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#728] [5. Q&A]
월드 1.3.0 입니다 

하려는것 

World.multiplay 폴더 안에 player_data.json파일을 넣고 불러오고 싶습니다 
![image](https://user-images.githubusercontent.com/50188795/167570342-73d62a00-788b-4ba7-bcd2-ee766e429c11.png)

player_data.json 파일은 그냥 json 파일이예요 

fs 모듈을 사용 안되드라구요

**Comment by AngeryOrang** [comment]
클라단에서 로드하고 첫 번째로 접속한 클라이언트가 패킷으로 보내주도록 짜야 할 것 같네요
혹은 index.ts에 string변수로 고스란히 데이터를 선언해야 할듯합니다.

**Reply by smilejsu82** [reply]
감사합니다 ~^^
